client/scripts/views/components/sidebar/index.ts

```diff
@@ -203,7 +204,8 @@ export const OnchainNavigationModule: m.Component<{}, {}> = {
           contentRight: [], // TODO
         }),
       // proposals (substrate, cosmos, moloch only)
-      !app.community && ((app.chain?.base === ChainBase.Substrate && app.chain.network !== ChainNetwork.Darwinia)
+      !app.community && ((app.chain?.base === ChainBase.Substrate && app.chain.network !== ChainNetwork.Darwinia
+                        && app.chain.network != ChainNetwork.HydraDX)
```

Comment:
Proposals page includes council motions, not just democracy. If Hydra has these, we'll want to make sure this link works.